types/jwt-express/index.d.ts

```diff
@@ -0,0 +1,222 @@
+// Type definitions for jwt-express 1.1
+// Project: https://github.com/AustP/jwt-express#readme
+// Definitions by: Nick Paddock <https://github.com/nickp10>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/// <reference types="node" />
+
+import express = require("express");
+import jsonwebtoken = require("jsonwebtoken");
+
+declare global {
```

Comment:
I don't think this declare global + nested namespace is doing the right thing. I think the intention is one of 2 things:

1. Give nice names to the types of the package. If this is the case, the names would be nicer just at the top-level as `export interface Request`, etc.
2. Merge with types from express. I don't think this will actually merge -- for this it has to be `declare module "express"` instead.

If (2) is the case, a test that shows actual usage on `Expression.Request.jwt` would be good.

Reply:
Yes, this is case #2. I was using the types for express-session as an example. I can make the changes you requested, but this does work as-is. You're right I forgot to include a test for this scenario and I will add that as well.